tests/test_views.py

```diff
@@ -19,9 +19,7 @@ def test_main_page_(self):
     def test_main_page_paginated_correctly(self):
         response = self.client.get(MAIN_PAGE_URL)
 
-        self.assertEqual(
-            len(response.context["post_list"]), PAGINATION
-        )
+        self.assertEqual(len(response.context["post_list"]), PAGINATION)
 
```

Comment:
The test `test_main_page_paginated_correctly` assumes that the number of posts in the database is greater than or equal to the pagination limit. This might not always be the case, especially if the test database is not set up correctly. Consider creating the necessary number of posts within the test itself to ensure that the pagination functionality can be tested properly.